packages/wonder-blocks-tooltip/src/components/tooltip-popper.tsx

```diff
@@ -44,10 +45,17 @@ type Props = {
      * on where there is available room within the document body.
      */
     rootBoundary?: RootBoundary;
+    /**
+     * If `rootBoundary` is `viewport`, this padding value is used to provide
+     * spacing between the popper and the viewport. If not provided, default
+     * spacing is applied.
```

Comment:
**suggestion:** How do you feel about specifying the default spacing value here? (e.g. 12px)

Reply:
I added it in! I was avoiding adding specific implementation details so there was one source of truth, but I could see how explicitly documenting the details would be useful! We will need to make sure to update the docs if the value ever changes 😄